ocfweb/context_processors.py

```diff
@@ -14,13 +14,19 @@
 
 
 def get_base_css_classes(request):
-    if request.resolver_match.url_name:
+    if not request.resolver_match:
+        # Need this for the 404 page
+        yield ''
+    elif request.resolver_match.url_name:
```

Comment:
Yeah, that works too if I get rid of the else case. I wasn't sure if `get_base_css_classes(request)` would cause `ocf_template_processor(request)` to error if I only returned `None`.